src/journal.c

```diff
@@ -775,23 +800,45 @@ html_url_requested_cb(GtkHTML *doc, const gchar *url, GtkHTMLStream *handle, gpo
     if (!path)
         return;
 
-    GnomeVFSResult result;
-    GnomeVFSHandle *vfs;
-    result = gnome_vfs_open(&vfs, path, GNOME_VFS_OPEN_READ);
+    GFile *ifile = g_file_new_for_path(path);
+
+    GError *error = NULL;
+    GFileInputStream *istream = g_file_read(ifile, NULL, &error);
+    if (NULL == istream)
+    {
+        g_warning("Failed to open HTML file for reading: %s", error->message);
```

Comment:
As I understand (in my PR for the journal dialog modernization), the html_url_requested_cb function is (only?) used for loading the logo. So it's not an HTML file.

I propose to simply drop "HTML" for all of these messages. I think it's enough to just say "file". Even better to include the actual file name if there is any risk it's unclear, but that could be considered a luxury item at the moment. (Maybe that's part of the error message from GIO?)

Reply:
I dropped HTML from the error messages as you suggested.